examples/ecs/main.tf

```diff
@@ -15,13 +15,14 @@ module "ceramic_ecs" {
 
   count = var.deployments
 
-  acm_certificate_arn            = data.aws_acm_certificate._3boxlabs_com.arn
+  acm_certificate_arn            = var.acm_certificate_arn
```

Comment:
I was trying to keep the code in sync with our main TF repo. I can currently run `terraform apply` from within the `app-elp` folder in the main repo, or from within the `examples/ecs` folder in this repo using the same `tfvars`.

To keep that consistency, I'll add back the code I removed to both places, the main repo and this one.

Reply:
Actually, I'll make them optionally configurable.

Reply:
In the future, I think we should make more/all of the other values also configurable, or even consider creating them within this Terraform instead of needing them to be created out-of-band.